com.termius.termius.yml

```diff
@@ -0,0 +1,79 @@
+app-id: com.termius.termius
+
+runtime: org.gnome.Platform
```

Comment:
does this use libraries that are only in the GNOME platform or would it work with Freedesktop.org 18.08?

Reply:
I do not know because I would like to start it and I can not :(

Reply:
pk-gtk-module error

Reply:
honestly I want a little help to finish it because I have a lot of trouble understanding the whole structure

Reply:
Since this is an Electron app, it probably needs to be based on the electron2 baseapp. This way it should work with FD.o runtime.